ci/fedora.sh

```diff
@@ -15,7 +15,7 @@ yum -y install \
 yum -y install libidn2-devel docbook5-style-xsl || true
 
 # supported since CentOS 7 (CentOS 6 don't have python 3.5 meson dependency)
-if [ "$(basename $0)" = "centos.sh" ]; then
+if [ "$(basename "$0")" = "centos.sh" ]; then
```

Comment:
Also this: protect space in PATH: again problem found by shellcheck, not a real problem. This script is used for CI. It can be used locally, but if anybody creates space in path, many things get broken. Again: this project has real issues, why wasting time with fixing artificial problems?